Composer/Dockerfile

```diff
@@ -31,4 +102,4 @@ COPY --from=build /src/Composer/packages/tools ./packages/tools
 ENV NODE_ENV "production"
 RUN yarn --production --frozen-lockfile --force && yarn cache clean
 WORKDIR /app/Composer
-CMD ["yarn", "start:server"]
+CMD ["yarn", "start:server"]
```

Comment:
Please configure your editor to respect our configuration in `.editorconfig` (`insert_final_newline` in this case).

Reply:
Use bash instead powershell in not win32 system and remove powershell install in docker.

Reply:
Add a editorConfig extension in vs to check this.